code/modules/mob/living/silicon/silicon_say.dm

```diff
@@ -1,27 +1,61 @@
 /mob/living/proc/robot_talk(message)
 	log_talk(message, LOG_SAY, tag="binary")
-	var/desig = "Default Cyborg" //ezmode for taters
+
+	var/designation = "Default Cyborg"
+	var/span_class = "binarysay"
+
 	if(issilicon(src))
-		var/mob/living/silicon/S = src
-		desig = trim_left(S.designation + " " + S.job)
-	var/message_a = say_quote(message)
-	var/rendered = "Robotic Talk, [span_name("[name]")] <span class='message'>[message_a]</span>"
+		var/mob/living/silicon/player = src
+		designation = trim_left(player.designation + " " + player.job)
+
+	if(isAI(src))
+		// AIs are loud and ugly
+		span_class = "binarysay binarysay--aisay"
+
+	var/quoted_message = say_quote(message)
+
 	for(var/mob/M in GLOB.player_list)
 		if(M.binarycheck())
 			if(isAI(M))
-				var/renderedAI = span_binarysay("Robotic Talk, <a href='?src=[REF(M)];track=[html_encode(name)]'>[span_name("[name] ([desig])")]</a> <span class='message'>[message_a]</span>")
-				to_chat(M, renderedAI, avoid_highlighting = src == M)
+				to_chat(
+					M,
+					"<span class='[span_class]'>\
```

Comment:
I don't think that the part before the message (`Robotic Talk, Maria(AI) states,`) should be big as well. It's not consistent with AI radio.

![image](https://user-images.githubusercontent.com/66640614/189776135-02538b70-931d-471d-994e-bc5c295a12cc.png)

Reply:
Sure!

Reply:
I think it would be fine if Robotic talk was an exception to this since they're always an exception anyways.